AliceO2Group/Bookkeeping
# COVERAGE-FAILURE-EXAMINATION [O2B-1126]/runs per data pass

Problem:
https://github.com/AliceO2Group/Bookkeeping/pull/1333 fails notoriously due to test `should successfully export filtered runs` (timeout) added in https://github.com/AliceO2Group/Bookkeeping/pull/1392
To examine failure of https://github.com/AliceO2Group/Bookkeeping/actions/runs/7840534241/job/21395280519?pr=1333
Strange, because https://github.com/AliceO2Group/Bookkeeping/pull/1356 succeeds, i
Run locally, tests also pass
Observation on GH actions:
It fails when:
     - run all tests
     - run all with `--grep Runs Overview Page`
It succeeds when:
     - run only export test: --grep `Runs Overview Page should successfully export filtered runs`
     - run a problematic test and one before
     - https://github.com/AliceO2Group/Bookkeeping/actions/runs/7841039202/job/21396644977?pr=1393 `--grep Runs Overview Page`, after replacing `reloadPage` by `goToPage`, but https://github.com/AliceO2Group/Bookkeeping/actions/runs/7841215075/job/21397157024?pr=1333 failed when running all, then it magically succeeds after adding this commit: https://github.com/AliceO2Group/Bookkeeping/pull/1333/commits/15f4ce942ce55d7ccd226edce30085a2d1d80235
